src/status_im/translations/fr.cljs

```diff
@@ -249,7 +262,7 @@
    :confirm                               "Confirmer"
    :transaction                           "Transaction"
    :unsigned-transaction-expired          "La transaction non signée a expiré"
-   :status                                "Statut"
+   :status                                "Status"
```

Comment:
It should be "Statut" previous version was correct (unless it's intended for Status.im)

Reply:
It's the app name